### DISCUSSION TITLE
Technical documentation for Noah-MP v4.0.1

### DISCUSSION THREAD

**Discussion by martina01natali** [NCAR/noahmp#142] [Q&A] [answered]
Hi community,
I'm a new user diving into the physics of the model, I have the original technical documentation file from 2011 (https://www.jsg.utexas.edu/noah-mp/files/Noah-MP_Technote_v0.2.pdf) (not sure about the Noah-MP version here but I would guess the 2.7) and of course the v5 (https://github.com/NCAR/noahmp/blob/master/docs/NoahMP_v5_technote.pdf).

I'm using Noah-MP 4.0.1 on LIS, does anyone know if a technical note with nice equations was published for this version? Thanks in advance.

**Comment by dmocko** [comment, marked as answer]
Hi @martina01natali 

I answered your question on the NASA LIS GitHub discussions here:
    https://github.com/NASA-LIS/LISF/discussions/1600

I will let NCAR folks correct me if there is indeed Noah-MP-4.0.1-specific documentation.

**Reply by martina01natali** [reply]
Thank you for your fast response! Cheers
